includes/bidding-functions.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Good call! c0c15265ff16a887bd8f5e251a20b2095cc8f3b7 restores the functions into the class file with simplified names, since they're now namespace there's no need for redundancy.

Reply:
8627cf4f4be6a598ec29ec40bcc08c749fbb0e34 fixes additional missing changes for `\Newspack_Ads\register_bidder`